### PR TITLE
adding Auth Providers to deprecations-additions-removals-compatibility

### PR DESCRIPTION
**Pull request by phil198**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1042.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1042.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 1]
```diff
@@ -85,6 +85,24 @@ RETURN a.name, a.year
 
 | Extension of the xref:clauses/load-csv.adoc#azure-cloud-storage[LOAD CSV] clause to allow loading CSV files from Azure Cloud Storage URIs.
 
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role=noheader]
+----
+CREATE USER bob
+SET AUTH 'externalProvderName' {
```
Comment (Hunterness) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]:
```suggestion
SET AUTH 'externalProviderName' {
```

**Review by Hunterness — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1042.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1042.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Review by Hunterness — CHANGES_REQUESTED** [review]
As discussed on slack, we also need to document the show command and privilege updates (probably a good idea to sync with the CIP so everything is documented).

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1042.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1042.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1042.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1042.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 2]
```diff
@@ -85,6 +85,44 @@ RETURN a.name, a.year
 
 | Extension of the xref:clauses/load-csv.adoc#azure-cloud-storage[LOAD CSV] clause to allow loading CSV files from Azure Cloud Storage URIs.
 
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role=noheader]
+----
+CREATE USER bob
```
Comment (Hunterness) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc:127]: should probably mention/show ALTER as well 🤔

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 3]
```diff
@@ -85,6 +85,44 @@ RETURN a.name, a.year
...
+----
+a|
+New support for `WITH AUTH` to allow display users' Auth Providers with a separate row per user per Auth Provider.
+
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role="noheader"]
+----
+[GRANT\|DENY] SET AUTH TO role
```
Comment (Hunterness) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]: might be nicer to do like we have for other privileges:
```
a|
label:functionality[]
label:new[] +
New privilege:
[source, cypher, role="noheader"]
----
ALTER DATABASE
----
a|
New privilege that allows a user to modify databases.
```
to not have this weird not-really-syntax thing

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 4]
```diff
@@ -85,6 +85,44 @@ RETURN a.name, a.year
...
+----
+a|
+New support for the `SET AUTH` privilege which allows the addition/removal of link:{neo4j-docs-base-uri}/operations-manual/{page-version}/authentication-authorization/auth-providers.adoc[Auth Providers] to/from users. This is a sub-privilege of the `ALTER USER` privilege. Like all `GRANT`/`DENY` commands this is only available in Enterprise Edition.
```
Comment (Hunterness) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]:
```suggestion
New support for the `SET AUTH` privilege which allows the addition/removal of link:{neo4j-docs-base-uri}/operations-manual/{page-version}/authentication-authorization/auth-providers.adoc[Auth Providers] to/from users.
This is a sub-privilege of the `ALTER USER` privilege.
Like all `GRANT`/`DENY` commands this is only available in Enterprise Edition.
```

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc (hunk 2 above)
Comment (phil198) [reply modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc:127]: done, thanks

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc (hunk 3 above)
Comment (phil198) [reply modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]: done, thanks

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc (hunk 4 above)
Comment (phil198) [reply modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]: done, thanks

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1042.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1042.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

File: modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc [hunk 5]
```diff
@@ -85,6 +85,60 @@ RETURN a.name, a.year
...
+Administration of the native (username / password) auth via the new syntax is also now supported (Community Edition).
+
+a|
+label:functionality[]
+label:new[]
+[source, cypher, role=noheader]
+----
+ALTER USER bob
+REMOVE AUTH 'native'
+----
```
Comment (Hunterness) [inline modules/ROOT/pages/deprecations-additions-removals-compatibility.adoc]:
```suggestion
REMOVE AUTH 'native'
SET AUTH 'externalProviderName' {
  SET ID 'userIdForExternalProvider'
}
----
```
just to have examples of both

**Review by Hunterness — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1042.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1042.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1042.surge.sh

Updated pages:
- [Deprecations, additions, and compatibility](https://neo4j-docs-cypher-1042.surge.sh/cypher-manual/5/deprecations-additions-removals-compatibility)

**Review by JPryce-Aklundh — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**